Tests fail on macOS if ghostscript isn't installed separately

Looking to contribute from macOS Ventura 13.5 (22G74), using `brew` for installing additional tools/dependencies. 

From the "first_steps.rst" file in `docs`, given [the macOS](https://github.com/Kozea/WeasyPrint/blob/master/docs/first_steps.rst#macos) section, I did `brew install weasyprint` thinking that the formulae would pull in any executable dependencies that weasyprint needed for development. 

In running the `venv/bin/python -m pytest` to confirm I had a stable install, I got 479 failed test:

![Screenshot 2023-08-02 at 4 06 50 PM](https://github.com/Kozea/WeasyPrint/assets/78614213/ad9c64f3-01fe-42a5-9c20-02868f0ac65f)
(including raw text output _tucked_ into Details to help with future searches) 

Using `brew install ghostscript` ([brew](https://formulae.brew.sh/formula/ghostscript)) resolved the "not found" errors and `gs` is found on the path. 

I'm open to pull requesting a note about this in `contribute.rst` if that seems valuable. 

I'm fine with this issue being closed as "won't fix" too. I hope this issue would be sufficient is how to resolve the problem if someone else run into it. 

🙇 

<details> 

## Raw Shell Output 

```
FAILED tests/draw/svg/test_transform.py::test_transform_scaley - FileNotFoundError: [Errno 2] No such file or directory: 'gs': 'gs'
FAILED tests/draw/svg/test_transform.py::test_transform_matrix - FileNotFoundError: [Errno 2] No such file or directory: 'gs': 'gs'
FAILED tests/draw/svg/test_transform.py::test_transform_multiple - FileNotFoundError: [Errno 2] No such file or directory: 'gs': 'gs'
FAILED tests/draw/svg/test_transform.py::test_transform_unknown - FileNotFoundError: [Errno 2] No such file or directory: 'gs': 'gs'
FAILED tests/draw/svg/test_units.py::test_units_px - FileNotFoundError: [Errno 2] No such file or directory: 'gs': 'gs'
FAILED tests/draw/svg/test_units.py::test_units_em - FileNotFoundError: [Errno 2] No such file or directory: 'gs': 'gs'
FAILED tests/draw/svg/test_units.py::test_units_ex - FileNotFoundError: [Errno 2] No such file or directory: 'gs': 'gs'
FAILED tests/draw/svg/test_units.py::test_units_unknown - FileNotFoundError: [Errno 2] No such file or directory: 'gs': 'gs'
FAILED tests/draw/svg/test_visibility.py::test_visibility_visible - FileNotFoundError: [Errno 2] No such file or directory: 'gs': 'gs'
FAILED tests/draw/svg/test_visibility.py::test_visibility_hidden - FileNotFoundError: [Errno 2] No such file or directory: 'gs': 'gs'
FAILED tests/draw/svg/test_visibility.py::test_visibility_inherit_hidden - FileNotFoundError: [Errno 2] No such file or directory: 'gs': 'gs'
FAILED tests/draw/svg/test_visibility.py::test_visibility_inherit_visible - FileNotFoundError: [Errno 2] No such file or directory: 'gs': 'gs'
FAILED tests/draw/svg/test_visibility.py::test_display_inline - FileNotFoundError: [Errno 2] No such file or directory: 'gs': 'gs'
FAILED tests/draw/svg/test_visibility.py::test_display_none - FileNotFoundError: [Errno 2] No such file or directory: 'gs': 'gs'
FAILED tests/draw/svg/test_visibility.py::test_display_inherit_none - FileNotFoundError: [Errno 2] No such file or directory: 'gs': 'gs'
FAILED tests/draw/svg/test_visibility.py::test_display_inherit_inline - FileNotFoundError: [Errno 2] No such file or directory: 'gs': 'gs'
FAILED tests/layout/test_inline.py::test_font_stretch - assert 101.546875 < 101.546875
FAILED tests/layout/test_table.py::test_table_vertical_align - FileNotFoundError: [Errno 2] No such file or directory: 'gs': 'gs'
=============================== 479 failed, 1442 passed, 38 xfailed in 134.12s (0:02:14) ===============================
(venv)  ✘ ✝  ~/repos/WeasyPrint   master  which gs
gs not found
```

</details>

--- a/docs/contribute.rst
+++ b/docs/contribute.rst
@@ -44,7 +44,7 @@ Tests
 Tests are stored in the ``tests`` folder at the top of the repository. They use
 the pytest_ library.
 
-Test required Ghostscript_ to be installed and available on the local path.
+Tests require Ghostscript_ to be installed and available on the local path.
 
 You can launch tests using the following command::
 

--- a/docs/contribute.rst
+++ b/docs/contribute.rst
@@ -44,6 +44,8 @@ Tests
 Tests are stored in the ``tests`` folder at the top of the repository. They use
 the pytest_ library.
 
+Test required Ghostscript_ to be installed and available on the local path.
+
 You can launch tests using the following command::
 
   venv/bin/python -m pytest
@@ -55,6 +57,7 @@ style::
   venv/bin/python -m flake8
 
 .. _pytest: https://docs.pytest.org/
+.. _Ghostscript: https://www.ghostscript.com/
 .. _isort: https://pycqa.github.io/isort/
 .. _flake8: https://flake8.pycqa.org/
 

